### DISCUSSION TITLE
Alpinejs $id property works incorrectly

### DISCUSSION THREAD

**Discussion by N1ebieski** [livewire/livewire#7575] [7. Bugs (archived)]
### Livewire version

3.3.1

### Laravel version

10.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

The version 3.3.1 of Livewire has changed a Alpinejs [magic property $id](https://alpinejs.dev/magics/id).

Until now, the property always returned a unique identifier > https://wirebox.app/b/4z10g but as of version 3.3.1 the same code returns:

item-1
item-1
item-1
item-1
item-1

I can't link to wirebox because it is not updated yet.

### Repository or Wirebox URL

https://wirebox.app/b/xvdvl

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@N1ebieski can you make your `wirebox` link public? As I can't access it at the moment. I've asked Philo to update wirebox to include the latest version so we can compare them. Or even better, could you submit a failing test PR so we can have a look at it? Thanks!

**Reply by N1ebieski** [reply]
@joshhanley here you go:

version 3.3.0 > https://wirebox.app/b/4z10g
version 3.3.3 > https://wirebox.app/b/xvdvl

**Reply by lrljoe** [reply]
Odd!  With a slight tweak, it does increment, but doesn't start where you'd expect
https://wirebox.app/b/40l7z

**Reply by N1ebieski** [reply]
It's not a solution because I need unique id for :key in x-for loop.

Of course I've found workaround > https://wirebox.app/b/4q2j2 but:

1. The names of the ids are still wrong: item-1-1, item-2-2 instead item-1-1, item-1-2
2. I think the _patch_ updates shouldn't change the behavior of existing methods
